### PR TITLE
Updating Grafana to 2.0.

### PR DESCRIPTION
**Pull request by vishh**
Tested this manually and it works.

### CODE REVIEW

**Comment by vishh** [comment]
cc @dchen1107

File: deploy/kube-config/influxdb/grafana-service.json [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-    "ports": [
-      {
-        "port": 80,
-        "targetPort": 8080
-      }
-    ],
-    "selector": {
-      "name": "influxGrafana"
+    "spec": {
+	"type": "LoadBalancer",
```
Comment (dchen1107) [inline deploy/kube-config/influxdb/grafana-service.json]: Indent here?

File: deploy/kube-config/influxdb/grafana-service.json [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
-  "apiVersion": "v1",
-  "kind": "Service",
-  "metadata": {
-    "labels": {
-      "kubernetes.io/name": "monitoring-grafana",
-      "kubernetes.io/cluster-service": "true"
+    "apiVersion": "v1",
+    "kind": "Service",
+    "metadata": {
+	"labels": {
```
Comment (dchen1107) [inline deploy/kube-config/influxdb/grafana-service.json]: indent?

**Comment by dchen1107** [comment]
Besides those nits, LGTM.

File: deploy/kube-config/influxdb/grafana-service.json (hunk 1 above)
Comment (vishh) [inline deploy/kube-config/influxdb/grafana-service.json]: The entire file was auto-indented. Do you want me to alter the indentation?

Comment (vishh) [inline deploy/kube-config/influxdb/grafana-service.json]: @dchen1107 can you clarify your comments?

**Pull request merged**